lib/cli/commands/placeorder.ts

```diff
@@ -4,7 +4,7 @@ import { PlaceOrderRequest, Order } from '../../proto/xudrpc_pb';
 
 export const command = 'placeorder <pair_id> <order_id> <quantity> [price]';
 
-export const describe = 'place an order';
+export const describe = 'place an order, if the price is 0 or unspecified -> market order';
```

Comment:
How about "place an order, if price is 0 or unspecified a market order is placed"